apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -175,6 +176,40 @@ export const DEFAULT_BLOCKS: Record<
     title: "Rick Astley - Never Gonna Give You Up",
     url: "https://www.youtube.com/embed/dQw4w9WgXcQ?si=ggGGn4uvFWAIelWD",
   },
+  // TODO: Replace with actual working API endpoint
+  [DYNAMIC_STATISTICS_TYPE]: {
+    type: `${DYNAMIC_STATISTICS_TYPE}`,
+    apiEndpoint: "https://jsonplaceholder.com/muis_prayers_time",
+    title: "hijri_date",
+    statistics: [
+      {
+        label: "Subuh",
+        key: "subuh",
+      },
+      {
+        label: "Syuruk",
+        key: "syuruk",
+      },
+      {
+        label: "Zohor",
+        key: "zohor",
+      },
+      {
+        label: "Asar",
+        key: "asar",
+      },
+      {
+        label: "Maghrib",
+        key: "maghrib",
+      },
+      {
+        label: "Ishak",
+        key: "isyak",
```

Comment:
in your sample json i saw each gregorian date has a corresponding arabic date. how is that obtained?

Reply:
thinking of it being referenced by the `title` field
 https://github.com/opengovsg/isomer/blob/709c2511a485b12cc24fbc2cfd09b6b9f7b48b50/apps/studio/src/components/PageEditor/constants.ts#L183